demo/src/react-admin/modules/content-page/services/content-page.service.ts

```diff
@@ -608,25 +604,25 @@ export class ContentPageService {
 		return description ? sanitizeHtml(description, sanitizePreset) : null;
 	}
 
-	public static async getContentPages(
-		filters: Partial<ContentTableState> | undefined,
-		page: number,
-		size: number,
-		orderProp: string,
-		orderDirection: 'asc' | 'desc'
-	): Promise<GraphQlResponse<ContentPageInfo[]>> {
-		const parsed = await dataService.query<ContentPageInfo[]>({
-			query: this.getQueries().GetContentPagesDocument,
-			variables: {
-				filters,
-				page,
-				size,
-				orderProp,
-				orderDirection,
-			},
-		});
-		return parsed;
-	}
+	// public static async getContentPages(
```

Comment:
correct that this is commented?

Reply:
Yes, i removed the commented code. The correct function to use is: fetchContentPages